app/jobs/after_transition_tasks_for_rejected_return_job.rb

```diff
@@ -2,7 +2,7 @@ class AfterTransitionTasksForRejectedReturnJob < ApplicationJob
   def perform(submission, transition)
     transition ||= submission.last_transition
 
-    submission.tax_return.transition_to(:file_rejected)
+    # submission.tax_return&.transition_to(:file_rejected) #ctc only
```

Comment:
this stops this: https://codeforamerica.sentry.io/issues/4748500712/?project=1880321&referrer=pivotal_plugin

having this in wouldn't impact this job but annoying none the less